projects/libreoffice/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -eu
+
+#shuffle CXXFLAGS -stdlib=libc++ arg into CXX as well because we use
+#the CXX as the linker and need to pass -stdlib=libc++ to build
+export CXX="$CXX -stdlib=libc++"
+#similarly force the -fsanitize etc args in as well as pthread to get
+#things to link successfully during the build
+export LDFLAGS="$CFLAGS -lpthread"
+
+cd $OUT
+#build under $OUT cause its the only place with enough space to do so
+#Filesystem              Size  Used Avail Use% Mounted on
+#/dev/mapper/docker...   10G   3.7G  6.4G  37% /
+#/dev/sda1               440G  316G  102G  76% /out
+#where $OUT is /out and $SRC and $WORK are on / which hasn't sufficient space
+#build in a sub scratch dir so we can easily throw away the build artifacts
+#afterwards
+mkdir -p scratch
+cd scratch
+$SRC/libreoffice/autogen.sh --with-distro=LibreOfficeOssFuzz --with-external-tar=$SRC
+
+make build-nocheck
+
+#copy in linked dependencies on build platform not in deployment platform
+cp /usr/lib/x86_64-linux-gnu/libfontconfig.so.1 instdir/program
```

Comment:
Do you copy them so that they won't be available to the fuzzer?
That won't work - ClusterFuzz doesn't support custom shared libraries. Libraries have to be linked it statically.